src/ArraySetops.chpl

```diff
@@ -0,0 +1,137 @@
+/* Array set operations
+ includes intersection, union, xor, and diff
+
+ currently, only performs operations with integer arrays 
+ */
+
+module ArraySetops
+{
+    use ServerConfig;
+
+    use BlockDist;
```

Comment:
I don't think BlockDist or CommAggregation are used in this module.

Reply:
You are right, thanks, fixed that